stripe-core/src/main/java/com/stripe/android/core/StripeAppInfo.kt

```diff
@@ -0,0 +1,13 @@
+package com.stripe.android.core
+
+import androidx.annotation.RestrictTo
+
+/**
+ * Setter for identifying your plug-in or library.
+ *
+ * See [Setting the API version](https://stripe.com/docs/building-plugins#setappinfo).
+ */
+object StripeAppInfo {
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
I'm struggling to understand why we'd restrict the field, but not the object. Again, I think even though it's small it could use an API review.